### ISSUE TITLE
'import' and 'export' may only appear at the top level (45:0) in process-es6/browser.js

### ISSUE THREAD

**Issue by iamale** [calvinmetcalf/rollup-plugin-node-globals#1]
Hello!

I get this weird error when I activate rollup-plugin-node-globals:
```
SyntaxError: 'import' and 'export' may only appear at the top level (45:0) in project_root/node_modules/rollup-plugin-node-globals/node_modules/process-es6/browser.js
    at Parser.pp.raise (project_root/node_modules/rollup-plugin-node-globals/node_modules/rollup-plugin-inject/node_modules/acorn/dist/acorn.js:943:13)
```

What could have gone wrong?

**Comment by calvinmetcalf** [comment]
you are using in conjunction with the commonjs plugin which is mistaking it for a commonjs file that needs to be transpiled, assuming you're just using process, you should add the ignoreGlobals flag as soon as the version that supports it is published.

**Comment by calvinmetcalf** [comment]
I believe v1.0.3 should work the current commonjs plugin, so you should be able to use that between now and the commonjs version getting updated

**Comment by iamale** [comment]
OK, thanks for the advices! Installing the git master version of `plugin-commonjs` worked too (I don't think it's a good solution for production, but I hope that a new release is coming out soon).

**Comment by tyler-johnson** [comment]
Any ETA on this?

Also 1.0.3 does not seems to work with `rollup-plugin-commonjs@2.2.1`. Is there another version combination I can try?

**Comment by calvinmetcalf** [comment]
I have no control nor knowledge of when their going to publish that, and 1.0.3 should work, what issue do you get?

**Comment by tyler-johnson** [comment]
It's the same issue:

```
'import' and 'export' may only appear at the top level (2:0) in /Users/tyler/Development/temple/node_modules/rollup-plugin-node-globals/src/global.js
```

However it could be an environment specific thing. I will try rearranging plugins to see what happens.

**Comment by tyler-johnson** [comment]
Although now I'm thinking that it is not the same issue since the file paths are different. It is very strange that my config file imports are pulling straight from the `src/` folder.

**Comment by calvinmetcalf** [comment]
yeah just put that file in your excludes from the commonjs plugin

**Comment by tyler-johnson** [comment]
Yep that was the fix! Thanks!
